lib/models/user.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'user.g.dart';
+
+// TODO: improve all of this if possible, using json:api adapter
```

Comment:
eventually, I will query this with graphql so, in the future, it will be simpler when I add graphQL client
![image](https://user-images.githubusercontent.com/13435717/112510244-d5cc3080-8dc3-11eb-9ded-2becef85feea.png)

Reply:
I can't wait to see this 💯